### PR TITLE
Full HD Window plus Fullscreen

### PR DESCRIPTION
**Pull request by fredrix96**
Test if the screen is displayed correctly and if fullscreen works by changing the variables in main.cc

### CODE REVIEW

**Review by kattalainen — DISMISSED** [review]

**Review by jakkeez — CHANGES_REQUESTED** [review]
Fix main.cpp in Game. Engine.Init has missing arguments like fullscreen, width and height

File: StortSpelProjekt/Engine/src/Renderer/Renderer.cpp [hunk 1]
```diff
@@ -1096,7 +1096,7 @@ void Renderer::initRenderTasks()
 		COMMAND_INTERFACE_TYPE::DIRECT_TYPE,
 		m_pBloomResources->GetPingPongResource(0),
 		m_pBloomResources->GetPingPongResource(1),
-		800, 600); // TODO: Send screen width/height from window after merge to develop
+		window->GetScreenWidth(), window->GetScreenHeight());
```
Comment (jocke1995) [inline StortSpelProjekt/Engine/src/Renderer/Renderer.cpp]: Nice!

File: StortSpelProjekt/Engine/src/Renderer/Renderer.cpp [hunk 2]
```diff
@@ -628,7 +628,7 @@ void Renderer::updateMousePicker()
 	}
 }
 
-void Renderer::initRenderTasks()
+void Renderer::initRenderTasks(const Window* window)
```
Comment (jocke1995) [inline StortSpelProjekt/Engine/src/Renderer/Renderer.cpp]: Isn't window saved as a member variable in Renderer?

File: StortSpelProjekt/Game/src/main.cpp [hunk 3]
```diff
@@ -6,9 +6,14 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int nCmdShow)
 {
     _CrtSetDbgFlag(_CRTDBG_ALLOC_MEM_DF | _CRTDBG_LEAK_CHECK_DF);
 
+	// Window values (will be influenced by the option class)
+	bool fullscreen = false;
+	int screenWidth = 1920;
+	int screenHeight = 1080;
```
Comment (jocke1995) [inline StortSpelProjekt/Game/src/main.cpp]: When you change this, you modify both the window size + the resolution of the renderTargets. Is this what you meant to do? This means that we will not be able to test the game in "high-res" without going fullscreen.

**Review by jocke1995 — CHANGES_REQUESTED** [review]
I left a few comments regarding some issues that I might not understand, or that you could possibly have missed.

File: StortSpelProjekt/Engine/src/Renderer/Renderer.cpp (hunk 2 above)
Comment (fredrix96) [reply StortSpelProjekt/Engine/src/Renderer/Renderer.cpp]: No?

File: StortSpelProjekt/Game/src/main.cpp (hunk 3 above)
Comment (fredrix96) [reply StortSpelProjekt/Game/src/main.cpp]: Yes, that is what I meant to do. However, you do have a valid point and I will add a resolutionWidth and a resolutionHeight as well

**Pull request closed without merging**